### ISSUE TITLE
NVDA does not read a list item until the end if you use parentheses at the end of a list item

### ISSUE THREAD

**Issue by milq** [nvaccess/nvda#14254]
With this code:

```html
<ol>
  <li>Qualification criteria: technical application (6 points), dexterity (2 points), presentation (2 points).</li>
</ol>
```

Actual behavior:

> NVDA Speech:
> "Qualification criteria: technical application (6 points), dexterity (2 points), presentation (2"

Expected behavior:

> NVDA Speech:
> "Qualification criteria: technical application (6 points), dexterity (2 points), presentation (2 points)."

**Comment by seanbudd** [comment]
Welcome @milq,

We are unable to process this issue as it stands.

- Please fill out the issue template fully. What browsers have you tested with? What versions of those browsers? What version of NVDA are you using?

Kind Regards,
NV Access Software Developers

**Comment by seanbudd** [comment]
What synthesizer are using?

**Comment by feerrenrut** [comment]
Hi, @milq in browse mode NVDA breaks lines at 100 characters (by default), it is likely that the end of this list item is on the next line, try pressing `down arrow` to read the next line.

In future, please fill out the issue template. The information requested is important for understanding the issue.

Given:
```html
<button>Before</button>
<ol>
  <li>Qualification criteria: technical application (6 points), dexterity (2 points), presentation (2 points).</li>
</ol>
<button>After</button>
```
NVDA output (copied from speech viewer, with speak command keys enabled)
```
shift+tab
Before  button  
down arrow
list  with 1 items  1. Qualification criteria: technical application (6 points), dexterity (2 points), presentation (2 
down arrow
points).
down arrow
out of list  button    After
```

Given this seems to be working as expected, I'm closing this issue.

**Comment by milq** [comment]
@seanbudd and @feerrenrut, thanks for your responses.

I'm sorry for the inconvenience I've caused. Next time I will fill in all the fields and try to be more precise.

Thank you for your work and for the great contribution you are making to the community.
